src/lib.rs

```diff
@@ -37,7 +37,7 @@ fn resolve_config_path(config_path: Option<PathBuf>) -> PathBuf {
                 path
             }
             None => {
-                err::abort_suggest_issue("Unable to find $HOME directory");
+                err::abort_suggest_issue(Box::new("Unable to find $HOME directory"));
```

Comment:
I tried removing `Box::new` from here and it compiles fine. Could we remove it then?
```suggestion
                err::abort_suggest_issue("Unable to find $HOME directory");
```

Reply:
Huh weird, I guess maybe the reference change made this possible.